ros2msg/package.xml

```diff
@@ -21,6 +21,10 @@
   <test_depend>std_msgs</test_depend>
   <test_depend>std_srvs</test_depend>
 
+  <deprecated>
```

Comment:
This is supposed to go in the `<export>` tag.

Reply:
Per REP 149: https://www.ros.org/reps/rep-0149.html#deprecated

Reply:
oops, fixed c12af9f